misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/main/java/de/metas/camel/externalsystems/shopware6/order/processor/OLCandRequestProcessor.java

```diff
@@ -326,4 +340,28 @@ private void processShopwareConfigs(
 		}
 		return groupsOptional;
 	}
+
+	@NonNull
+	private Optional<JsonOLCandCreateRequest> processTax(
+			@Nullable final TaxProductIdProvider taxProductIdProvider,
+			@NonNull final JsonOLCandCreateRequest.JsonOLCandCreateRequestBuilder olCandCreateRequestBuilder,
+			@NonNull final JsonTax tax)
+	{
+		if (taxProductIdProvider == null)
+		{
+			return Optional.empty();
+		}
+
+		return Optional.of(
+				olCandCreateRequestBuilder
+						.externalLineId(FREIGHT_COST_EXTERNAL_LINE_ID_PREFIX + tax.getTaxRate())
```

Comment:
> .externalLineId(FREIGHT_COST_EXTERNAL_LINE_ID_PREFIX + tax.getTaxRate())
I'm not sure if this is the best solution... but I'm 100% sure we need an external line Id in order to avoid creating an ol cand twice for the freight cost ( in the scenario the same order shopware is considered multiple times for import )